### PR TITLE
add updates for ReportSchedule which only takes a workspace now

### PR DESCRIPTION
**Pull request by AaronLasseigne**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -3288,6 +3321,8 @@ definitions:
         type: boolean
       org_seat_limit_reached:
         type: boolean
+      python_performance_first_tier:
```
Comment (msalli) [inline swagger.yaml:3324]: What's all the new stuff for, or why is it here?

Comment (AaronLasseigne) [reply swagger.yaml:3324]: Good catch. Maybe someone updated and didn't push their changes?

Comment (AaronLasseigne) [reply swagger.yaml:3324]: Yup, they're recent representer updates.

Comment (msalli) [reply swagger.yaml:3324]: Okay, nice nice.

**Review by msalli — APPROVED** [review]
⏩

**Review by BitPuncher — APPROVED** [review]

**Pull request merged**